pytorch_lightning/trainer/training_loop.py

```diff
@@ -474,6 +473,12 @@ def run_training_epoch(self):
             if early_stop_epoch or self.fast_dev_run:
                 break
 
+        if not self.is_overriden('validation_step'):
+            self.call_checkpoint_callback()
```

Comment:
is it not a problem that this could get called twice now that you moved this down here? So basically the case "should_check_val=True" and "val_step not overridden". Is that fine?

Reply:
true, I ll fix it...